storage/aptosdb/src/backup/test.rs

```diff
@@ -19,12 +18,12 @@ proptest! {
     fn test_get_transaction_iter(input in arb_blocks_to_commit()) {
         let tmp_dir = TempPath::new();
         let db = AptosDB::new_for_test(&tmp_dir);
-        let mut smt = SparseMerkleTree::<StateValue>::default().freeze();
-
+        let mut in_memory_state = (*db.state_store.in_memory_state().lock()).clone();
+        let _ancester = in_memory_state.current.clone().freeze();
```

Comment:
nit: name: "_frozen" (it holds a ref to the ancester, but still reflects the current state)